spec/controllers/admin/tickets_controller_spec.rb

```diff
@@ -0,0 +1,247 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe Admin::TicketsController do
+  let(:admin) { create(:admin) }
+  let(:conference) { create(:conference) }
+  let!(:ticket) { create(:ticket, conference: conference) }
+  let(:new_title) { Faker::Hipster.sentence }
+
+  context 'admin is signed in' do
+    before { sign_in admin }
+
+    describe 'GET #index' do
+      before { get :index, conference_id: conference }
```

Comment:
This now needs to be updated, but that's pretty quick.

https://github.com/snap-cloud/snapcon/commit/231b2a6d7215e9b330572cc1ef3fa2746a217c2d

Merged this in my fork and it's working well!